documentation/src/main/markdown/currentreleasenotes.md

```diff
@@ -9,6 +9,8 @@
 * npm 6, which was deprecated in [solution_name] 8.x, is no longer supported.
 * [solution_name] 7.x has entered end of support. See the [Product Maintenance, Support, and Service Schedule page](https://sig-product-docs.synopsys.com/bundle/blackduck-compatibility/page/topics/Support-and-Service-Schedule.html) for further details.
 * (IDETECT-3879) The detectors\[N\].statusReason field of the status.json file will now contain the exit code of the detector subprocess command in cases when the code is non-zero.
+* The Gradle wrapper directory `gradle` will be excluded from signature scan by default, in addition to node_modules, bin, build, .git, .gradle, out, packages, target. Use
```

Comment:
Perhaps switch this sentence up a bit to:
"In addition to node_modules, bin, build, .git, .gradle, out, packages, target, the Gradle wrapper directory `gradle` will be excluded from signature scan by default."

Reply:
Done, thanks.